server/model/monitor.js

```diff
@@ -447,6 +456,18 @@ class Monitor extends BeanModel {
                         options.data = bodyValue;
                     }
 
+                    if (this.cacheBust) {
+                        let cacheBust = "";
+                        const characters = "abcdefghijklmnopqrstuvwxyz0123456789";
+                        const charactersLength = characters.length;
+                        for (let i = 0; i < 6; i++) {
+                            cacheBust += characters.charAt(Math.floor(Math.random() * charactersLength));
+                        }
```

Comment:
a shorter way would be the following:

```suggestion
                        const randomFloatString = Math.random().toString(36);
                        const cacheBust = randomFloatString.substring(2);
```

For equivalence (except in `cacheBurst` length) see 
![image](https://github.com/louislam/uptime-kuma/assets/26258709/7cf1b261-2b74-414c-af4e-7ff29ff7e91b)

Reply:
Btw, you as the author can resolve discussions, making this clearer to other reviewers what was addressed and what is an open discussion ;)